pkg/image/registry/imagestreamtag/rest.go

```diff
@@ -333,11 +333,14 @@ func newISTag(tag string, imageStream *imageapi.ImageStream, image *imageapi.Ima
 			Name:              istagName,
 			CreationTimestamp: event.Created,
 			Annotations:       map[string]string{},
+			Labels:            imageStream.Labels,
```

Comment:
I had originally put annotations in here as well, but realized that we already have annotations set and backed off.  I don't think we want labels per image stream tag - the stream controls the labels, and unless we split tags into their own objects (sometime in the future), I think we shouldn't lie about what the labels are.